src/components/cta/ExternalsAuth.tsx

```diff
@@ -102,34 +133,63 @@ export const ExternalsAuth: React.FunctionComponent<ExternalsAuthProps> = ({
         })
     }
 
-    return authProvider === 'github' ? (
-        <Link
-            href="https://sourcegraph.com/.auth/github/login?pc=https%3A%2F%2Fgithub.com%2F%3A%3Ae917b2b7fa9040e1edd4&redirect=/get-cody"
-            className={classNames(
-                `btn hover:sg-bg-hover-external-auth-button flex items-center px-4 hover:text-black md:h-12 md:px-6 md:text-lg
+    let link: React.ReactElement;
+    link =  <Link href="" />;
+
+    if (authProvider === 'github') {
+        link = (
+            <Link
+                href="https://sourcegraph.com/.auth/github/login?pc=https%3A%2F%2Fgithub.com%2F%3A%3Ae917b2b7fa9040e1edd4&redirect=/get-cody"
+                className={classNames(
+                    `btn hover:sg-bg-hover-external-auth-button flex items-center px-4 hover:text-black md:h-12 md:px-6 md:text-lg
                  ${dark ? 'hover:btn-primary bg-black text-white ' : 'btn-inverted-primary text-black'}`,
-                className
-            )}
-            onClick={handleOnClick}
-            id="githubButton"
-        >
-            <GithubIcon className="pr-2" />
-            {label}
-        </Link>
-    ) : (
-        <Link
-            href="https://sourcegraph.com/.auth/gitlab/login?pc=https%3A%2F%2Fgitlab.com%2F%3A%3A262309265ae76179773477bd50c93c7022007a4810c344c69a7371da11949c48&redirect=/get-cody"
-            className={classNames(
-                `btn hover:sg-bg-hover-external-auth-button flex items-center px-4 hover:text-black md:h-12 md:px-6 md:text-lg ${
-                    dark ? 'sg-gitlab-bg-color hover:btn-primary text-white ' : 'btn-inverted-primary text-black'
-                }`,
-                className
-            )}
-            onClick={handleOnClick}
-            id="gitlabButton"
-        >
-            <GitlabColorIcon className="pr-2" />
-            {label}
-        </Link>
-    )
+                    className
+                )}
+                onClick={handleOnClick}
+                id="githubButton"
+            >
+                <GithubIcon className="pr-2" />
+                {label}
+            </Link>
+        );
+    }
+
+    if (authProvider === 'google') {
+        link = (
+            <Link
+                href="https://sourcegraph.com/.auth/openidconnect/login?pc=google"
+                className={classNames(
+                    `btn hover:sg-bg-hover-external-auth-button flex items-center pl-3 pr-3 hover:text-black md:h-12 md:text-lg 
+                    ${dark ? 'sg-gitlab-bg-color hover:btn-primary text-white ' : 'btn-inverted-primary text-black' }`,
+                    className
+                )}
+                onClick={handleOnClick}
+                id="googleButton"
+            >   
+                <GoogleColorIcon className="pr-4" />
+                <span className="pr-2">{label}</span>
+            </Link>
+        );
+    }
+
+    if (authProvider === 'gitlab') {
```

Comment:
Nit: Other places you had these in alphabetical order, but here Google is before GitLab. Consider keeping them in a consistent order.